release_notes/ocp-4-4-release-notes.adoc

```diff
@@ -808,3 +827,20 @@ must also link to the build config object.
 the service catalog. The `controller-manager` and `apiserver` Pods show a status
 of `CrashLoopBackOff`.
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1821589[*BZ#1821589*])
+
+* Installing {rh-openstack} 16 with Kuryr does not work due to the
+`ovn_controller` crashing after initial setup.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1812009[*BZ#1812009*])
+
+* The Red Hat Virtualization (RHV) machine `instance-state` annotation and the
+`providerStatus.instanceState` status do not always match. This causes the
+client to fail or incorrectly patch the RHV machine status.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1815394[*BZ#1815394*])
+
+* When scaling up a MachineSet on RHV, the new machine gets stuck in the
+Provisioned phase. This causes the machine to never run.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1815435[*BZ#1815435*], link:https://bugzilla.redhat.com/show_bug.cgi?id=1817853[*BZ#1817853*])
+
+* Applying autoscaling to an {product-title} cluster on RHV fails due to
```

Comment:
Something like: "{product-title} cluster autoscaling on RHV fails due to cluster resource computation errors."

Unclear who or what is applying autoscaling as-is.